src/test/java/org/opensearch/neuralsearch/ml/MLCommonsClientAccessorTests.java

```diff
@@ -59,15 +63,34 @@ public void testInferenceSentence_whenValidInput_thenSuccess() {
             actionListener.onResponse(createModelTensorOutput(TestCommonConstants.PREDICT_VECTOR_ARRAY));
             return null;
         }).when(client).predict(Mockito.eq(TestCommonConstants.MODEL_ID), Mockito.isA(MLInput.class), Mockito.isA(ActionListener.class));
+        setupMocksForTextEmbeddingModelAsymmetryCheck(false);
 
-        accessor.inferenceSentence(TestCommonConstants.MODEL_ID, TestCommonConstants.SENTENCES_LIST.get(0), singleSentenceResultListener);
+        accessor.inferenceSentence(
```

Comment:
I understand you are not using the old `inferenceSentence` method. Can you double check whether this method is still need? We don't wish to see redundant function.